src/models/@Mdl_TwoTrailersCLASS/SymbolicComputationOfEoM.m

```diff
@@ -0,0 +1,74 @@
+function SymbolicComputationOfEoM()
+%SYMBOLICCOMPUTATIONOFEOM Summary of this function goes here
+%   Detailed explanation goes here
+    syms distance Lt1 L2 Lt2 L3 slip_right slip_left dt
+
+    p = [distance, Lt1, L2, Lt2, L3, slip_right, slip_left];
+
+    nx = 9;
+
+    nu = 2;
+
+    % State = [x_i, y_i, theta_i]
+    x = sym("x", [nx, 1]);
+
+    % Input = [v_r, v_l]
+    u = sym("u", [nu, 1]);
+
+    % State = [x_i, y_i, theta_i]
+    state = sym("x", [nx, 1]);
+
+    % Input = [v_r, v_l]
+    input = sym('u', [nu,1]);
+
+    %% Motion model
+
+    v = ((1 - slip_right) * u(1) + (1 - slip_left) * u(2))/2;
+
+    w = ((1 - slip_right) * u(1) - (1 - slip_left) * u(2))/(2*distance);
+
+    theta_12 = x(3) - x(6);
+
+    theta_23 = x(6) - x(9);
```

Comment:
For joint angle between trailer and trailer, gamma is used. 
theta is for robot heading